# ComputeRef.srv
---
float64 x_ref
float64 y_ref
float64 psi_ref